libs/ledger-live-common/src/walletconnect/index.ts

```diff
@@ -52,47 +49,24 @@ export const parseCallRequest: Parser = async (account, payload) => {
 
     // @dev: Today, `eth_signTypedData` is versionned. We can't only check `eth_signTypedData`
     //       This regex matches `eth_signTypedData` and `eth_signTypedData_v[0-9]`
+    // https://docs.metamask.io/guide/signing-data.html
     case payload.method.match(/eth_signTypedData(_v.)?$/)?.input:
       message = JSON.parse(payload.params[1]);
-      hashes = {
```

Comment:
Are you sure the hashes are not used by wallet connect / the sign message flow?
They seem to be used and displayed [here](https://github.com/LedgerHQ/ledger-live/blob/feat/LIVE-3180-EIP-712/apps/ledger-live-desktop/src/renderer/components/SignMessageConfirm/index.jsx#L66-L89) on LLD for example

Reply:
Ah yes, you're right.
I missed this one. I was not sure, this was also the purpose of a pre-review :)